frontend/src/components/AboutDialog/AboutDialog.css

```diff
@@ -0,0 +1,10 @@
+.AboutLogoDiv {
```

Comment:
can this be replaced by just assigning the class Flex? or using some combo of classes we already have defined in App.css?